release-tools/desktop-cluster/start-cluster.sh

```diff
@@ -0,0 +1,223 @@
+#!/bin/bash
+# This script starts an Elasticsearch and Kibana cluster from an ODFE tarball. It un-tars the tarball, creates sensible elasticsearch.yml and kibana.yml files, starts the two processes, and waits for them to become available.
+
+set -e
+
+function usage() {
+    echo "Usage: $0 [options] -e /path/to/elasticsearch.tgz -k /path/to/kibana.tgz"
+    echo "Or   : $0 [options] -v <version>"
+    echo "Mandatory arguments:"
+    echo "-e PATH\tPath to the elasticsearch tarball. You must specify both -e and -k, or use -v to download an ODFE version."
+    echo "-k PATH\tPath to the kibana tarball. You must specify both -e and -k, or use -v to download an ODFE version."
+    echo "-v VERSION\tDownload ODFE version <VERSION>. This overrides the -e and -k options. If the files 'odfe-<VERSION>.tar.gz' or 'odfe-kibana-<VERSION>.tar.gz' exist in the current directory then they will not be re-fetched."
+    echo ""
+    echo "Options:"
+    echo "-u\tRun unattended. If this is not specified, then the script will wait for the user to hit 'return' and automatically terminate the elasticsearch and kibana processes."
+    echo "-w\tSet the working directory. This directory will be deleted and recreated. If not specified, then a new random temporary directory will be used. The tarballs will be unpacked into this directory; logs and the process pid files will be stored here."
+    echo "-a\tSet the maximum number of times to probe the elasticsearch and kibana processes to see if the services are up. Defaults to 12."
+    echo "-b\tSet the backoff (in seconds) between probes to the elasticsearch and kibana processes to see if they are up. Defaults to 10."
+    echo "-h\tPrint this help message."
+}
+
+MAX_ATTEMPTS=12
+BACKOFF=10
+
+while getopts ":he:k:w:uv:a:b:" arg; do
```

Comment:
```
:he:k:w:uv:a:b: or :h:e:k:w:u:v:a:b:
```

Reply:
The -h and -u options do not require arguments.

Reply:
Forgot about that. Thanks.